backend/entities/Views/Property.cs

```diff
@@ -117,6 +118,7 @@ public class Property
         /// <summary>
         /// get/set - The most recent estimate.
         /// </summary>
+        [Column(TypeName = "MONEY")]
```

Comment:
Is it possible to create a constant/enum or something for the values of the TypeName?